Orleans.Sagas/SagaState.cs

```diff
@@ -16,7 +15,5 @@ public class SagaState
         public bool HasBeenAborted { get; set; }
 
         public Dictionary<string, string> Properties { get; set; }
-
-        public Dictionary<Type, SagaError> Errors { get; set; }
```

Comment:
Can we now remove the `SagaError` class completely?

Reply:
You are right, missed it

Reply:
No need, as we mark the function as obsolete